xaitk_saliency_demo/app/engine/options.py

```diff
@@ -14,24 +14,28 @@
         ],
         # Task => input
         "input_expected": 2,
+        # Better defaults:
+        "proximity_metric": "euclidean",
     },
     "detection": {
         # Task => saliency
-        "saliency_active": "detection-saliency",
+        "saliency_active": "DRISEStack",
         "saliency_available": [
-            {"text": "Default", "value": "detection-saliency"},
+            {"text": "DRISE", "value": "DRISEStack"},
+            {"text": "RandomGridStack", "value": "RandomGridStack"}
         ],
         # Task => model
-        "model_active": "DetectionFasterRCNN",
+        "model_active": "DetectionFRCNN",
         "model_available": [
-            {"text": "Faster R-CNN", "value": "DetectionFasterRCNN"},
-            {"text": "RetinaNet", "value": "DetectionRetinaNet"},
+            {"text": "FRCNN (COCO)", "value": "DetectionFRCNN"},
+            {"text": "CenterNet (VisDrone)", "value": "DetectionCenterNetVisdrone"},
         ],
         # Task => input
         "input_expected": 1,
         # Better defaults:
         "n": 200,
-        "proximity_metric": "cosine",
+        "p1": 0.05,
+        "proximity_metric": "euclidean",
```

Comment:
> I will have to play around with this a bit more, but seems strange to use Euclidean here when the original D-RISE paper used cosine.

The inclusion of proximity metric here as a "parameter override" is actually an error because we removed the specification of custom metrics from our D-RISE implementation specifically due to the original paper using cosine only. I will remove this line entirely.

Aside: these "better params" overrides are effectively not doing anything right now. I am not sure if that is intentional in a re-work that may have previously happened.

> Seems like the root issue may be that we got rid of the saliency auto-normalization option ...

Correct. I was not sure what to do about this at this stage. I am open to suggestion, but I think that is a separate branch/PR. This package/functionality update did not introduce this (it's observable on main with the equivalent algorithm choice).

> Somewhat related to this, we may want to update the screenshots in the gallery ...

I can update that here or that can be a separate PR.

Reply:
Ok, let's leave the saliency auto-normalization and updating gallery pics for a different PR.

As you stated, it looks like it's safe to remove the detection "better params" completely (these [lines](https://github.com/XAITK/xaitk-saliency-web-demo/blob/3e1757b3d42bca28382fb0ed514d660d731e1ee7/xaitk_saliency_demo/app/engine/options.py#L35-L38)). Feel free to do that and then merge this PR. Thanks!

Reply:
:+1: